include/utils.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef OBC_UTILS_HPP
+#define OBC_UTILS_HPP
+
+#include <type_traits>
+
+namespace obc {
+
+template <typename Enum>
+constexpr auto to_underlying(Enum e)
+{
+    return static_cast<std::underlying_type_t<Enum>>(e);
+}
+
+}  // namespace obc
+
+#endif
```

Comment:
skoro to jest używane tylko w stosunku do resultów to czemu tego nie wrzucić do result.cpp? a w ogóle to jest jedno rzutowanie to po co robić taki wrapper na to?

Reply:
a to akurat jest bardzo przydatna rzecz i spora szansa jest ze sie jeszcze nam przyda. Najlepiej ją wydzielic do czesci bibliotecznej